source/io/module_propertyNode.js

```diff
@@ -30,48 +30,48 @@
         NO_ERROR: {
             CODE: 0,
             MESSAGE: ''
+        },
+
+        kNIObjectReferenceIsInvalid: {
+            CODE: 1055,
+            MESSAGE: 'Object reference is invalid.'
         }
     };
 
-    /* var formatMessageWithException = function (messageText, exception) {
+    var formatMessageWithException = function (messageText, exception) {
         if (typeof exception.message === 'string' && exception.message.length !== 0) {
             return messageText + ', Additional information: ' + exception.message;
```

Comment:
I think this is supposed to be `<APPEND>\n` instead of `Additional Information` see the following: https://github.com/ni/VireoSDK/blob/master/source/io/module_httpClient.js#L112

Reply:
I found this in module_JavaScriptInvoke.js should I modify that one too?